bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -63,38 +66,57 @@ Sets the timeintervall(seconds) between consecutive readings.
 - Steps or Granularity (set in `.archives=<consolidationfunction>,<xff>,<steps>,<rows>`
 
 Steps are the number of consecutive readings that are used the create a single entry into the database for this timeintervall.
-The timeintervall covered is calculated by (step x step) seconds.
+The timeintervall covered is calculated by (step x steps) seconds.
```

Comment:
Please replace all occurrences of `timerintervall` with `time interval`. I see 3.